adobe/spectrum-web-components
# fix(textfield): textarea actually grows with multiline

## Description


sp-textarea was not growing when typing multiple lines of text, so I updated the`height` of textarea with `scrollHeight`

## Related issue(s)
#4259 


-

## Motivation and context


It updates the height of textarea with attributes {multiline,grows} when typing multiple lines of text.

## How has this been tested?




    1. Go [here](https://opensource.adobe.com/spectrum-web-components/components/textarea/#grows)
    2. Remove any text from the middle of the textarea
    3. Start adding text and press Enter
    4. Check the textarea doesn't grow 

## Screenshots (if appropriate)

https://github.com/adobe/spectrum-web-components/assets/32033946/5ba144e0-c84a-4fde-951f-b29a79f1af0d



## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
